cura/Arranging/ArrangeObjectsAllBuildPlatesJob.py

```diff
@@ -147,6 +147,8 @@ def run(self):
         status_message.hide()
 
         if not found_solution_for_all:
-            no_full_solution_message = Message(i18n_catalog.i18nc("@info:status", "Unable to find a location within the build volume for all objects"),
-                                               title = i18n_catalog.i18nc("@info:title", "Can't Find Location"))
+            no_full_solution_message = Message(i18n_catalog.i18nc("@info:status",
+                                                                  "Unable to find a location within the build volume for all objects"),
+                                               title = i18n_catalog.i18nc("@info:title", "Can't Find Location"),
+                                               message_type = Message.MessageType.ERROR)
```

Comment:
I think this is a warning. It will load the object, it just wont give it a place.